CHANGELOG.md

```
@@ -33,12 +47,33 @@
+- **#3:** fix Ci checks ([77cfd5a](https://github.com/VilnaCRM-Org/website/commit/77cfd5adcbc7b03e3eff272d38308459f047bc50))
+- **#3:** fix memory-leak test ([eae62ce](https://github.com/VilnaCRM-Org/website/commit/eae62ceda1736025a374e1792f98d41df9ebe661))
+- **#3:** return run dev in memory-leak ([415e71e](https://github.com/VilnaCRM-Org/website/commit/415e71e29995d67d7e88d97122db26e219834ac3)), closes [#3](https://github.com/VilnaCRM-Org/website/issues/3)
+- **#5:** fix memory-leak ([71ae934](https://github.com/VilnaCRM-Org/website/commit/71ae934952951c734015088096a77d19a5c1e26a))
+- **#5:** fix pnpm-lock ([cb8a59a](https://github.com/VilnaCRM-Org/website/commit/cb8a59aabc5b19e896ba51d5e2069b8c5913046c))
+- **#5:** update package.json ([9571993](https://github.com/VilnaCRM-Org/website/commit/95719934e7c46633ce5613a330d50f21fe52f04f)), closes [#5](https://github.com/VilnaCRM-Org/website/issues/5)
+- Dockerfile to reduce vulnerabilities ([ff7ec8e](https://github.com/VilnaCRM-Org/website/commit/ff7ec8ef830bc40aa5f5747050f1e872b26e7941))
+- fix autorelease.yml ([deb1231](https://github.com/VilnaCRM-Org/website/commit/deb1231770ad9aeb1171e0f35ab1860e91c4cffb))
+
+### Features
+
+- **#12:** add engines to package.json ([e093282](https://github.com/VilnaCRM-Org/website/commit/e093282ca694c628ec30aeaacbbdd4731b8fb283)), closes [#12](https://github.com/VilnaCRM-Org/website/issues/12)
```

markdownlint-cli2 (0.17.2)

Line length
Expected: 100; Actual: 204

(MD013, line-length)